lib/quicksilver-config-client.js

```diff
@@ -51,6 +53,21 @@ class QuicksilverConfigClient {
   getExchangeSettings() {
     return true;
   }
+
+  /**
+   * Check that configuration file define in .env as QS_CONFIG_PATH is accessable.
+   */
+  configFileFound() {
+    return new Promise((resolve) => {
+      fs.access(this.qsConfigPath, fs.F_OK, (err) => {
```

Comment:
In addition, this wolud return true even if `this.qsConfigPath` is a directory. It'd be better to [`fs.stat`](https://nodejs.org/api/fs.html#fs_fs_stat_path_callback).